src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4262,7 +4266,16 @@ .FORWARDHELPCATEGORY Cmdlet
     }
     else
     {
-        $help | more
+        # Respect PAGER, use more on Windows, and use less on Linux
+        if (Test-Path env:PAGER) {
+            $moreCommand = (Get-Command -CommandType Application $env:PAGER | Select-Object -First 1).Definition
```

Comment:
This will lose the ability to specify arguments to the pager utility - `$pager,$moreArgs = $env:PAGER -split '\s+' `.   That doesn't bother me personally but presumably somebody added these for a reason (an issue I suspect).

Reply:
Yes, that was explicitly to fix https://github.com/PowerShell/PowerShell/issues/6142

Should just use the same code as from https://github.com/PowerShell/PowerShell/pull/6144/files

Reply:
Thanks for the reminder! 

Fixed.

Reply:
Is it possible to add tests to exclude such regression?